README.md

```diff
@@ -1 +1,27 @@
-# ccd-docker-definition-importer
\ No newline at end of file
+# ccd-docker-definition-importer
+
+Docker image to load definitions into CCD. Definitions are excel files which are retrieved from urls passed to the image comma-separated in an environment variables.
+
+
+## Configuration
+
+The scripts that load the definitions expect the following environment to be available.
+
+| Parameter                  | Description                                | Default  |
+| -------------------------- | ------------------------------------------ | -------- |
+| `CCD_DEF_URLS`        | URLs to retrieve the definition xls files comma-separated. The script terminates if this is empty. | `` |
```

Comment:
could maybe format this table with https://marketplace.visualstudio.com/items?itemName=josa.markdown-table-formatter ?